requirements/cross-chain-transfer.md

```diff
@@ -10,57 +10,59 @@ Various scenarios around these interactions between users and federation members
 
 ### The cross chain transfer passes chain B's maximum reorg length
 
-Finally, as chain B progresses and the transfer's maturity passes the maximum reorg length, all federation nodes monitoring the B chain can update the status of the corresponding session in their respective stores. Once the status of the session is marked complete, no more work will be required for this session.
+Finally, as chain B progresses and the transfers' maturities pass the maximum reorg length, all federation nodes monitoring the B chain can update the statuses of the corresponding transfer sessions in their respective stores. Once the statuses of the sessions are marked complete, no more work will be required for these sessions.
 
 ![The cross chain transfer matures on targeted chain](../assets/cross-chain-transfers/happy-path-4.svg)
 
 ## 2) Preferred leader offline
 
-### A cross chain transfer is triggered while leader is offline
+### The cross chain transfers are triggered while leader is offline
 
-Similar to the previous case, deposit transactions to the federation's multisig on chain A have triggered a cross chain transfer. Here however, the preferred leader happens to be offline. All other nodes carry on as usual, creating and exchanging partially signed transactions. However, no one is here to broadcast it as FM<sub>2</sub> is offline.
+Similar to the previous case, deposit transactions to the federation's multisig on chain A have triggered some cross chain transfers. Here however, the preferred leader happens to be offline. All other nodes carry on as usual, creating and exchanging partially signed transactions. However, no one is here to broadcast it as FM<sub>2</sub> is offline.
 
 ![A cross chain transfer is triggered while leader is offline](../assets/cross-chain-transfers/leader-offline-1.svg)
 
-### The cross chain is handled one block later by next leader
+### The cross chain transfers are handled one block later by next leader
 
-As time passes, a new block appears on chain A, and a new leader is elected (FM<sub>3</sub>). This new leader can now broadcast the transaction and we are back on the normal path (cf 1.).
+As time passes, a new block appears on chain A, and a new leader is elected (FM<sub>3</sub>). This new leader can now broadcast the transactions and we are back on the normal path (cf 1.).
 
 As described earlier, every new leader must first check for previously unprocessed transfers and process those first, before attending to the transfers on the current block. If a few leaders are offline, this can create a backlog of transfers.
-If a leader finds a previously unprocessed transfer (and they don't have enough signatures to broadcast) they must do nothing and wait for the other members to send over signatures.
+If a leader finds a previously unprocessed transfers (and they don't have enough signatures to broadcast) they must do nothing and wait for the other members to send over signatures.
```

Comment:
I think we may need to clarify this - do transactions always get processed in the order they appear on chain A? The federation can very quickly come into disagreement about which UTXOs to use if we do not enforce this, but that does mean that a single transaction can halt the entire transfer process.